tests/tests.rs

```diff
@@ -309,6 +309,32 @@ macro_rules! test_header {
 // This file is generated by build.rs
 include!(concat!(env!("OUT_DIR"), "/tests.rs"));
 
+#[test]
+fn test_clang_env_args() {
+    std::env::set_var("BINDGEN_EXTRA_CLANG_ARGS", "-std=c++03");
```

Comment:
This test is not ok, since tests run in parallel, it will affect the output of other random tests (as seen on Travis).

I don't have other brilliant ideas about how to test this though... Maybe something like:

```
std::env::set_var("BINDGEN_EXTRA_CLANG_ARGS", "-DDEFINED_FROM_ENV");
```

on `bindgen-integration/build.rs`, and then something like:

```
#ifndef DEFINED_FROM_ENV
#error "BINDGEN_EXTRA_CLANG_ARGS is not working properly"
#endif
```

From one of the headers included there? Or conditionally defining something and using it from one of the `#[test]` functions?